front/src/routes/ApplicantsPage.svelte

```diff
@@ -80,6 +81,39 @@
     }));
     loading = false;
   });
+
+  let selectedCommittee = null;
+
+  const passesCommitteeFilters = (
+    rows: any[],
+    selectedCommittee: CommitteeType
+  ) => {
+    if (selectedCommittee == null) {
+      return rows;
+    }
+    return rows.filter(function (row) {
+      if (row.status == ApplicationStatus.ACCEPTED) {
+        return row.committee_accepted == selectedCommittee;
+      }
+      let commExists = false;
+      row.committees.forEach(function (commObj: { committee: CommitteeType }) {
+        if (commObj.committee == selectedCommittee) {
+          commExists = true;
+        }
+      });
+      return commExists;
```

Comment:
There is a basic approach to finding if something exists in the array, but you can also use built-in JS function called `some` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some), so you would do something like this
```suggestion
      return row.committees.some(({ committee }) => committee === selectedCommittee);
```